migrations/20200812162046_add_training_pils_table.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('training_pils')
+    .createTable('training_pils', table => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+      table.uuid('training_course_id').references('id').inTable('training_courses').notNull();
+      table.string('firstName').notNull();
+      table.string('lastName').notNull();
+      table.string('email').notNull();
+      table.date('dob').notNull();
```

Comment:
I think this will need to associate to profiles rather than duplicating fields, and then we will create profiles for users when they're added to courses (and a profile does not exist)

Reply:
yeah, I think you're right 👍